src/source/PeerConnection/SessionDescription.c

```diff
@@ -163,6 +165,8 @@ STATUS setPayloadTypesFromOffer(PHashTable codecTable, PHashTable rtxTable, PSes
     PCHAR fmtp;
     UINT64 fmtpScore, bestFmtpScore;
 
+    CHK(codecTable != NULL && rtxTable != NULL && pSessionDescription != NULL, STATUS_NULL_ARG);
```

Comment:
It's not immediately clear whether `codecTable` and `rtxTable` should be null checked here since they are not ever dereferenced in this `setPayloadTypesFromOffer` function. The check for `pSessionDescription` should definitely be here since it is dereferenced here and will segfault if it is null. Disallowing the other two pointers from being null, on the other hand, has the potential for changing intended behavior. I don't see that it would change the behavior in this case, but we might want to consider a less impactful change for this PR.

Reply:
This is an internal SDK function (not publicly accessible) and also this function is only used in 1 place: in `setRemoteDescription`.

The method uses both hash tables, and is replacing the elements inside of them:

`hashTableContains(codecTable), hashTableGet(codecTable), hashTableUpsert(codecTable), and hashTableUpsert(rtxTable)` 

Even though hashTable* methods have their own null check, I think it doesn't make sense to allow a null hash table as it will bail the method anyway once it calls that method. (`STATUS_NULL_ARG` will be returned regardless if I added this check or not, but this check makes it immediately obvious that we don't want a null hash table.)

We can revisit making the condition stronger in a future PR, as both tables look to have additional requirements which need some more investigating.

Reply:
> I think it doesn't make sense to allow a null hash table as it will bail the method anyway once it calls that method

I agree in this case - although it could change the method behavior, the null check is appropriate here.